### PR TITLE
 Bug 1175815 - [Contacts][NGA] Remove 'curtainFrame' dependency when matching

### PR DESCRIPTION
**Pull request by jpruden92**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=70344682cafc4e11994d7989def1fcd7639ddcb8

File: apps/communications/contacts/js/match_service.js [hunk 1]
```diff
@@ -9,6 +9,13 @@
   var extensionFrame = document.createElement('iframe');
   extensionFrame.id = 'match-extension';
   document.body.appendChild(extensionFrame);
+
+  if (document.querySelector('#fb-curtain') == null) {
```
Comment (borjasalguero) [inline apps/communications/contacts/js/match_service.js]: `if (!document...)`

Comment (borjasalguero) [inline apps/communications/contacts/js/match_service.js]: Create a bug in order to remove all 'fb-'  prefix and add a TODO with bug number, just to keep tracking of all pending tasks to do.

Comment (borjasalguero) [inline apps/communications/contacts/js/match_service.js]: Why dont we move this logic to https://github.com/mozilla-b2g/gaia/blob/master/shared/pages/import/js/curtain.js#L9 ? Seems that there is a collision there.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=8626e72f28a53172fbc6c0c0fa08e41407aeeb10

File: shared/pages/import/js/curtain.js [hunk 2]
```diff
@@ -6,6 +6,13 @@ var Curtain = (function() {
 
   var _ = navigator.mozL10n.get;
 
+  if (!parent.document.querySelector('#fb-curtain')) {
```
Comment (borjasalguero) [inline shared/pages/import/js/curtain.js]: Add a bug to remove this `#fb-` prefix (sync with @mancas about this) and add a `TODO` comment in order to remove all these dependencies as next step.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b9b7cbc6e4d0ea3973007876d422b5c99310550f

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58d005f917b0f7fc60cc5e72e67e7163144b8073

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=58d005f917b0f7fc60cc5e72e67e7163144b8073

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=b06466d4511f71eaa1864a6f4c24cb4f88edc169

**Pull request merged**